modules/router/src/admin/index.ts

```diff
@@ -100,12 +100,9 @@ export class AdminHandlers {
           path: ConduitString.Required,
           target: ConduitString.Required,
           action: ConduitString.Required,
-          description: ConduitString.Optional,
+          routeDescription: ConduitString.Optional,
```

Comment:
But why?

Reply:
Because it's a ConduitModel, and it disallows the use of "description" as a reserved word.